### PR TITLE
pretty little pet

### PR DESCRIPTION
**Pull request by digbijaya**
image

### CODE REVIEW

File: test_pet.py [hunk 1]
```diff
@@ -19,28 +21,51 @@ def test_pet_schema():
...
 TODO: Finish this test by...
 1) Extending the parameterization to include all available statuses
 2) Validate the appropriate response code
 3) Validate the 'status' property in the response is equal to the expected status
 4) Validate the schema for each object in the response
 '''
-@pytest.mark.parametrize("status", [("available")])
+
+'''
+QA_NOTE: Empty records returned for "sold" status but it is returning 200.
```
Comment (automationExamples) [inline test_pet.py:34]: good callout, this is intended for empty lists with a valid status

File: test_pet.py [hunk 2]
```diff
@@ -19,28 +21,51 @@ def test_pet_schema():
...
 2) Parameterizing the test for any edge cases
 '''
-def test_get_by_id_404():
-    # TODO...
-    pass
\ No newline at end of file
+
+
+'''
+QA OBSERVATION: The test is passing for "a_str" as path.
```
Comment (automationExamples) [inline test_pet.py:61]: good callout, i would consider this a bug, a different response is appropriate here

File: test_store.py [hunk 3]
```diff
@@ -12,5 +13,58 @@
...
+def get_order():
+    test_endpoint = "/pets/findByStatus"
+    place_order_endpoint = "/store/order"
+    params = {
+        "status": "available"
+    }
+    response = api_helpers.get_api_data(test_endpoint, params)
+    if len(response.json()) == 0:
+        pet_id_list = [pet_obj["id"] for pet_obj in api_helpers.get_api_data("/pets/").json()]
+        new_pet_id = max(pet_id_list) + 3
```
Comment (automationExamples) [inline test_store.py:33]: interesting approach!
